src/components/Icons/HeroBackground.js

```diff
@@ -0,0 +1,1812 @@
+      />
+      <path
+        id='prefix__XMLID_198_'
+        d='M606.5 208.5c.1 2.5.5 6.9.5 6.9 5.4.5 11.7-2.5 18.7-3.4 0 0 2 1.6-3.5 2.3'
+        className='prefix__st2'
+      />
+      <path
+        id='prefix__XMLID_197_'
+        d='M577.7 238.2l28.8-13c1.3-.5 2.2-1.6 2.4-3l.1-6.7'
+        className='prefix__st2'
+      />
+      <path
+        id='prefix__XMLID_196_'
+        d='M597.3 181.6s12.4.1 19.7 4.8c8 5.2 9.1 26.2 9.1 26.2'
+        className='prefix__st2'
+      />
+      <path
+        id='prefix__XMLID_195_'
+        d='M609.5 256.6s-4.1-6.9-2.3-22'
+        className='prefix__st3'
+      />
+      <path
+        id='prefix__XMLID_194_'
+        d='M565.2 220s.4 13.1-.3 16.7'
+        className='prefix__st2'
+      />
+      <path
+        id='prefix__XMLID_193_'
+        d='M692.1 52.1v106.8'
+        className='prefix__st4'
+      />
+      <path
+        id='prefix__XMLID_192_'
+        d='M538.6 254.9s7.1 1.3 7.3 9'
+        className='prefix__st3'
+      />
+      <path
+        id='prefix__XMLID_191_'
+        d='M560.4 258.2s2.6-7.1 3-10.4'
+        className='prefix__st2'
+      />
+      <circle
+        id='prefix__XMLID_190_'
+        cx={267.2}
+        cy={175.3}
+        r={2.9}
+        className='prefix__st3'
+      />
+      <g id='prefix__XMLID_120_'>
+        <path
+          id='prefix__XMLID_27_'
+          d='M472.3 63.4c-4.3.8-4 3.7-3.7 4.5.2.6.9 1 1.5 1.3 1.8.8 3.9 1.2 5.6.4 1.8-.8 2.8-3.2 1.7-4.8-1-1.6-3.2-1.6-5.1-1.4'
+          className='prefix__st6'
+        />
+        <g id='prefix__XMLID_22_'>
+          <path
+            id='prefix__XMLID_26_'
+            d='M472.2 62.9c-2 .4-4.1 1.5-4.2 3.8-.2 2.6 2.7 3.6 4.9 3.8 2.6.3 5.6-1.3 5.5-4.3-.1-3.1-3.6-3.5-6-3.3-.6.1-.6 1.1 0 1 1.8-.2 5.1-.2 5 2.5-.1 2.9-3.5 3.5-5.7 2.8-1.4-.4-3-1.2-2.6-2.8.4-1.6 2-2.2 3.4-2.5.6-.2.3-1.1-.3-1z'
+            className='prefix__st5'
+          />
+        </g>
+      </g>
+      <path
+        id='prefix__XMLID_119_'
+        d='M509.9 103.9s-.1 8.7 1 10.6c0 0 11-.6 11.6-2.6 0 0-2.3-6.7-2.1-9.5l-10.5 1.5z'
+        className='prefix__st4'
+      />
+      <path
+        id='prefix__XMLID_113_'
+        d='M548.5 103.9s-.1 8.7 1 10.6c0 0 11-.6 11.6-2.6 0 0-2.3-6.7-2.1-9.5l-10.5 1.5z'
+        className='prefix__st4'
+      />
+      <path
+        id='prefix__XMLID_95_'
+        d='M526.7 103s-.1 9.6 1.1 11.5c0 0 11-.6 11.6-2.6 0 0-2.3-6.7-2.1-9.5l-10.6.6z'
+        className='prefix__st24'
+      />
+      <path
+        id='prefix__XMLID_89_'
+        d='M499.6 120.2s-.1 9.6 1.1 11.5c0 0 11-.6 11.6-2.6 0 0-2.3-6.7-2.1-9.5l-10.6.6z'
+        className='prefix__st24'
+      />
+      <path
+        id='prefix__XMLID_5_'
+        d='M227.8 389.8h-96.6'
+        className='prefix__st2'
+      />
+      <path
+        id='prefix__XMLID_4_'
+        d='M60.9 389.8h46.3'
+        className='prefix__st2'
+      />
+      <path id='prefix__XMLID_3_' d='M38.1 389.8h16' className='prefix__st2' />
+    </g>
+  </svg>
+);
+
```

Comment:
Po dluzszym namysle, zoptymalizyjmy to i wjebmy jakos img src, bo z 1800 linii nam sie zjebie text to html ratio